addons/planner/static/src/less/planner.less

```diff
@@ -204,6 +204,172 @@
         margin-left: auto;
         margin-right: auto;
 
+        .trophy{
```

Comment:
I'm ok with this deplacement, but try to reduce the less. Remove some attributes, even if it change a little the complet design. 700 lines of less, AL will not be happy ^^